client/packages/system/src/Stock/api/hooks/line/useStockLines.ts

```diff
@@ -1,13 +1,9 @@
-import { useQuery, useUrlQueryParams } from '@openmsupply-client/common';
-import { useStockApi } from '../utils/useStockApi';
+import { useQuery } from '@openmsupply-client/common';
+import { ListParams, useStockApi } from '../utils/useStockApi';
 
-export const useStockLines = () => {
+export const useStockLines = (queryParams: ListParams) => {
   const api = useStockApi();
 
-  const { queryParams } = useUrlQueryParams({
```

Comment:
Moved this to a higher level since was hard to tell why the location.name filter wasn't working

Reply:
we've moved this out in a few hooks now, I think it's much clearer